arangod/Cluster/ClusterInfo.cpp

```diff
@@ -5394,6 +5291,20 @@ void ClusterInfo::shutdownSyncers() {
   _curSyncer->beginShutdown();
 }
 
+
+void ClusterInfo::waitForSyncersToStop() {
+  auto start = std::chrono::steady_clock::now();
```

Comment:
does it make sense here to notify the two Syncers, just in case they missed their `beginShutdown` call?

Reply:
There is no need to notify them. You will not win any insights. We're shutting down. So, no